ui_framework/components/color_picker/color_picker.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { ChromePicker } from 'react-color';
+
+export class KuiColorPicker extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showColorSelector: false,
+    };
+  }
+
+  closeColorSelector = () => {
+    this.setState({ showColorSelector: false });
+  };
+
+  toggleColorSelector = () => {
+    this.setState({ showColorSelector: !this.state.showColorSelector });
+  };
+
+  handleColorSelection = (color) => {
+    this.props.onChange(color.hex);
+  };
+
+  render() {
+    const color = this.props.color || '#ffffff';
```

Comment:
Oh I see! Hm. I just read up on what the [React team thinks about this](https://github.com/facebook/react/issues/2166), and here's what I'm thinking:

* What happens if `null` is provided? I'd expect either nothing to happen, or an error. In either case, I wouldn't expect null to be interpreted as white. If it's an error, then should we just surface that error instead of hiding it?
* I think we can set `color: PropTypes.string.isRequired` to have it default to white in the case that it's undefined.
* Then we can get rid of the check within the `render` method.

Thoughts on this? It just feels a bit brittle and unexpected to have these two checks in different places.